codeforamerica/vita-min
# Fix homepage banner and closing soon message dates

## Link to pivotal/JIRA issue
The banner on the homepage message and also the closing soon text message and email have hard coded dates that should be coming from the config dates that have been set up. [Jira ticket](https://codeforamerica.atlassian.net/browse/GYR1-4)

## What was done?
The dates in the banner were taken from the config file instead of being hard coded, the date for the April 1st document deadline was added to the config file and the closing soon messages now all have the dates coming in from the config instead of being hard coded 

## How to test?
Test banner when open for intake in order to make sure the dates are correct still and also test closing soon messages
